app/controllers/admin/email_controller.rb

```diff
@@ -88,6 +88,18 @@ def incoming
     render_json_dump(serializer)
   end
 
+  def incoming_from_bounced
+    params.require(:id)
+
+    bounced = EmailLog.find(params[:id].to_i)
```

Comment:
This is going to raise an error if the ID is invalid. We probably want to use `find_by` and raise `Discourse::InvalidParameters` instead so that the error message is clearer.